### DISCUSSION TITLE
How to get payload object from response in redirectHandler.

### DISCUSSION THREAD

**Discussion by bilik-tomas** [jiripudil/Naja#357] [Support]
Is here any way to get a payload object from response in redirectHandler ?

**Comment by jiripudil** [comment]
Hi, I don't think this is currently possible. What's your use case? Perhaps we'll find a way to get it there, or a workaround.

**Comment by bilik-tomas** [comment]
I was trying to backport force-redirect functionality to presenter payload object.

**Reply by jiripudil** [reply]
I see. Well, you can always bypass Naja's redirection handling and process it however you need in your own `success` event listener. Either send the target URL in a different payload key than "redirect", or call `event.preventDefault()` in the `redirect` event handler.
